.circleci/sck_values.yaml

```diff
@@ -0,0 +1,33 @@
+global:
+  splunk:
+    hec:
+      token:
+      host:
+      port: 8088
+      insecureSSL: true
+      protocol: http
+splunk-kubernetes-objects:
+  splunk:
+    hec:
+      indexName: sck-all-objects
```

Comment:
do we need indexName values here? since we are adding it from update_configurables_sck_artifacts.sh.